packages/keychain/src/profiles.ts

```diff
@@ -102,22 +120,25 @@ const sendUsernameToRegistrar = async ({
   zoneFile,
   identity,
 }: SendToRegistrarParams) => {
-  const { registerUrl } = registrars[subdomain];
+  const { registerUrl, apiKey, addressVersion } = registrars[subdomain];
 
   const registrationRequestBody = JSON.stringify({
     name: username,
-    owner_address: identity.address,
+    owner_address: c32check.b58ToC32(identity.address, addressVersion),
     zonefile: zoneFile,
   });
 
-  const requestHeaders = {
+  const headers = new Headers({
     Accept: 'application/json',
     'Content-Type': 'application/json',
-  };
+  });
+  if (apiKey) {
+    headers.set('Authorization', `bearer ${apiKey}`);
```

Comment:
```suggestion
    headers.set('Authorization', `Bearer ${apiKey}`);
```
Isn't this typically capitalised?

Reply:
Typically, yeah. But the registrar expects lowercase 🤷🏼 .